snap/snapcraft.yaml

```diff
@@ -226,9 +226,10 @@ parts:
     override-stage: |
         craftctl default
         pkg_version=$(
-            dpkg-deb -f \
-            $CRAFT_PART_SRC/../stage_packages/ovn-host*.deb \
-            Version | sed -rne 's/([0-9.]+)[-+].*$$/\1/p')
+            apt-cache policy ovn-host | \
```

Comment:
The current code is getting the version from the actual binary consumed by the build, the new version gets the version from some cache, which may or may not be in sync with the actually binary used.

Why do you think this change is necessary, can we keep the existing method?